src/svc/invocable-scripts/paths/scripts_scriptID.yml

```
@@ -22,24 +26,54 @@ get:
         application/json:
           schema:
             $ref: '../schemas/Script.yml'
-    '401':
-      $ref: '../../../common/responses/AuthorizationError.yml'
-    '500':
-      $ref: '../../../common/responses/InternalServerError.yml'
+          examples:
+            $ref: '../schemas/ScriptGetResponse.yml'
+    "401":
+      $ref: "../../../common/responses/AuthorizationError.yml"
+    "500":
+      $ref: "../../../common/responses/InternalServerError.yml"
+    '404':
+      description: |
+        Not found.
+      content:
+        application/json:
+          schema:
+            $ref: "../../../common/schemas/Error.yml"
+          examples:
+            notFound:
+              summary: |
+                The requested script was not found.
+              value: {
+                "code":"not found",
+                "message":"script \"09afa3b220fe400\" not found"
+              }
     default:
-      description: Unexpected error.
+      description: Internal server error.
       $ref: '../../../common/responses/ServerError.yml'
 patch:
   operationId: PatchScriptsID
   tags:
     - Invokable Scripts
   summary: Update a script
   description: |
-    Updates a [script](https://docs.influxdata.com/influxdb/cloud/api-guide/api-invokable-scripts/) and returns the script.
+    Updates an invokable script.
 
-    Use this endpoint to update the properties (`name`, `description`, and `script`) of an invokable script.
+    Use this endpoint to modify values for script properties (`description` and `script`).
+
+    To update a script, pass an object that contains the updated key-value pairs.
+
+    #### Limitations
+
+    - If you send an empty request body, the script will neither update nor
+    store an empty script, but InfluxDB will respond with an HTTP `200` status
+    code.
+
+    #### Related Guides
+
+    - [Invoke custom scripts](https://docs.influxdata.com/influxdb/cloud/api-guide/api-invokable-scripts/)
   requestBody:
-    description: The script update to apply.
+    description: |
+      An object that contains the updated script properties to apply.
```

```suggestion
      In the request body, provide the script properties to update.
```
This is my new pattern. Eventually, I plan to append an statement (with redoc-cli) that introduces the properties that can be set, like in YouTube docs--"You can set the following properties for this resource:"